h/accounts/test/views_test.py

```diff
@@ -32,6 +32,8 @@ def configure(config):
     config.registry.registerUtility(SubmitForm, IProfileForm)
     config.registry.registerUtility(MagicMock(), IRegisterSchema)
     config.registry.registerUtility(MagicMock(), IRegisterForm)
+    config.registry.feature = MagicMock()
```

Comment:
> Ok that line is actually crashing on master currently (but there's not test failing).

On master: login and choose Account from the username dropdown menu in the top right:

```
File "/home/seanh/Projects/hypothesis/h/h/accounts/views.py", line 308, in profile
    if request.registry.feature('notification'):
AttributeError: 'Registry' object has no attribute 'feature'
```